17-ea/jdk/ubi/ubi8/Dockerfile.open.releases.full

```diff
@@ -0,0 +1,142 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+# Artifactory credentials to download CRIU binary
+# Set your Artifactory credentials here or pass them at build time
+ARG ARTIFACTORY_TOKEN
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN dnf install -y \
+    # CRIU dependencies
+        iptables-libs jansson libibverbs libmnl libnet libnftnl libpcap nftables protobuf-c \
+    # Semeru dependencies
+        tzdata openssl curl ca-certificates fontconfig glibc-langpack-en gzip tar \
+    # other dependencies
+        python3 \
+    && dnf update -y; dnf clean all;
+
+LABEL name="IBM Semeru Runtime EA" \
+      vendor="International Business Machines Corporation" \
+      version="17.0.6.6" \
+      release="17" \
+      run="docker run --rm -ti <image_name:tag> /bin/bash" \
+      summary="IBM Semeru Runtime EA Docker Image for OpenJDK with openj9 and ubi" \
+      description="For more information on this image please see https://github.com/ibmruntimes/semeru-containers/blob/master/README.md"
+
+# Install CRIU
+RUN set -eux; \
+    ARCH="$(uname -m)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
+          CRIU_URL='https://na.artifactory.swg-devops.com/artifactory/sys-rt-generic-local/hyc-runtimes-jenkins.swg-devops.com/build-scripts/criu_build/3/criu.tar.gz'; \
+          CRIU_SHA_URL='https://na.artifactory.swg-devops.com/artifactory/sys-rt-generic-local/hyc-runtimes-jenkins.swg-devops.com/build-scripts/criu_build/3/criu.tar.gz.sha256.txt'; \
+         ;; \
+       *) \
+         echo "Unsupported arch: ${ARCH}"; \
+         exit 1; \
+         ;; \
+    esac; \
+    cd /tmp; \
+    curl -H X-JFrog-Art-Api:${ARTIFACTORY_TOKEN} -LfsSo criu.tar.gz ${CRIU_URL};\
+    curl -H X-JFrog-Art-Api:${ARTIFACTORY_TOKEN} -LfsSo criu.tar.gz.sha256.txt ${CRIU_SHA_URL};\
+    sha256sum -c criu.tar.gz.sha256.txt; \
+    tar -xzf criu.tar.gz --strip-components=1; \
+    cp -R usr/local /usr; \
+    echo /usr/local/lib64 > /etc/ld.so.conf.d/criu.conf; \
+    ldconfig; \
+    setcap cap_checkpoint_restore,cap_sys_ptrace=eip /usr/local/sbin/criu; \
+    rm -fr criu criu.tar.gz;
+
+ENV PATH="/usr/local/sbin:$PATH"
+
+ENV JAVA_VERSION 17.0.6.6
+
+RUN set -eux; \
+    ARCH="$(uname -m)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
+         ESUM='8ead64de8d19a360eb819afd57cdc0b40eef23edeb9e2aef05036801cd2fd3c7'; \
+         BINARY_URL='https://github.com/ibmruntimes/semeru17-ea-binaries/releases/download/jdk-17.0.6%2B6_december_22-preview_1/ibm-semeru-open-ea-jdk_x64_linux_17.0.6_6_December_22-preview_1.tar.gz'; \
+         ;; \
+       *) \
+         echo "Unsupported arch: ${ARCH}"; \
+         exit 1; \
+         ;; \
+    esac; \
+    curl -LfsSo /tmp/openjdk.tar.gz ${BINARY_URL}; \
+    echo "${ESUM} */tmp/openjdk.tar.gz" | sha256sum -c -; \
+    mkdir -p /opt/ibm/java-ea; \
+    cd /opt/ibm/java-ea; \
```

Comment:
Its good to use /opt/java folder instead of /opt/ibm to keep it in sync with the other OE Docker files. This change applies to the JRE file as well.